doc/reference/getting_started.rst

```diff
@@ -155,7 +155,7 @@ main Intel Open source EC FW documentation.
 .. code-block:: bash
 
    cd ../ecfwwork/zephyr_fork
-   git am ../../ecfw-zephyr/zephyr_patches/patches_v3.2.patch
+   git am ../../ecfw-zephyr/zephyr_patches/patches_v3.4.patch
```

Comment:
The file is named [patches_v3.4.patch] (https://github.com/intel/ecfw-zephyr/blob/opensource_update_v3.4/zephyr_patches/patches_v3.4.patch)
Maybe we could rename the file to match documentation and other similar files